wzl/src/composables/useDataSource.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file 处理表格数据
+ */
+
+import { type TableColumn, type TableProps } from "../components/types";
```

Comment:
```suggestion
import type { TableColumn, TableProps } from "../components/types";
```

`type` 类型建议统一写到外部，避免每次都写一下。

Reply:
fixed